spec/unit/intercom/request_spec.rb

```diff
@@ -7,27 +7,27 @@
   it 'raises an error when a html error page rendered' do
     response = OpenStruct.new(:code => 500)
     req = Intercom::Request.new('path/', 'GET')
-    proc {req.parse_body('<html>something</html>', response)}.must_raise(Intercom::ServerError)
+    _(proc {req.parse_body('<html>something</html>', response)}).must_raise(Intercom::ServerError)
```

Comment:
Thanks for this. Do you have happen to have a link to Minitest's documentation that outlines this behaviour?

Reply:
Ah, I see you've refactored this later on. Nice! 👍